plugins/utils/mini.nix

```diff
@@ -82,9 +85,13 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin {
   callSetup = false;
   hasSettings = false;
   extraConfig = cfg: {
-    extraConfigLua = lib.foldlAttrs (lines: name: config: ''
-      ${lines}
-      require(${lib.nixvim.toLuaObject "mini.${name}"}).setup(${lib.nixvim.toLuaObject config})
-    '') "" cfg.modules;
+    extraConfigLua =
+      lib.foldlAttrs (lines: name: config: ''
+        ${lines}
+        require(${lib.nixvim.toLuaObject "mini.${name}"}).setup(${lib.nixvim.toLuaObject config})
+      '') "" cfg.modules
+      + lib.optionalString ((lib.hasAttr "icons" cfg.modules) && cfg.mockDevIcons) ''
+        MiniIcons.mock_nvim_web_devicons()
```

Comment:
Q: where does the `MiniIcons` name come from? I didn't notice us importing it, i.e. using `require()`.

Reply:
Right above, we will do a require for `mini.icons` when the user supplies it in the attribute set. That's why I need to make sure that they added it before I call that.